public/app/features/dashboard/components/PanelEditor/VisualizationTab.tsx

```diff
@@ -50,6 +50,16 @@ export const VisualizationTabUnconnected: FC<Props> = ({ panel, plugin, changePa
           <Input
             value={searchQuery}
             onChange={e => setSearchQuery(e.currentTarget.value)}
+            onKeyPress={e => {
+              if (e.key === 'Enter') {
+                const query = e.currentTarget.value;
+                const plugins = getAllPanelPluginMeta();
+                const match = filterPluginList(plugins, query);
+                if (match && match.length) {
+                  onPluginTypeChange(match[0]);
+                }
```

Comment:
@dprokop -- not sure the best FC version of this... is it OK to have this inline?

Reply:
useCallback :) Pushed update